test/fixtures/output/kafka/partition/describe.golden

```diff
@@ -1,6 +1,6 @@
 +--------------+-----------+
-| Cluster      | cluster-1 |
+| Cluster      | lkc-12345 |
```

Comment:
This is an ID in some examples, and a name in others?

Reply:
`cluster-1` is an id, not a name, so these should all be ids.